test/unit/cart.js

```diff
@@ -11,8 +11,19 @@ import ShopifyBuy from '../../src/buybutton';
 let cart;
 
 describe('Cart class', () => {
+  let closeCartSpy;
+
   beforeEach(() => {
-    cart = new Cart({}, {
+    closeCartSpy = sinon.spy();
+    cart = new Cart({
+      options: {
+        cart: {
+          viewData: {
+            test: 'test string',
+          },
```

Comment:
For completeness, I think adding populated `contents` and `text` objects will better validate the `viewData` unit tests.